src/sphinxcontrib/towncrier/__init__.py

```diff
@@ -85,6 +85,19 @@ def _get_changelog_draft_entries(
     return towncrier_output
 
 
+def _get_towncrier_config(
```

Comment:
On a side note, it's probably best to move this to a submodule called `_towncrier` and keep all the shims there (but just this function, for now, I'll move the rest of related things separately). Also, this will allow reverting that WPS202 ignore.